src/pages/messages/components/NviCorrectionList.tsx

```diff
@@ -0,0 +1,15 @@
+import { useTranslation } from 'react-i18next';
+
+export const NviCorrectionList = () => {
+  const { t } = useTranslation();
+
+  return (
+    <iframe
```

Comment:
Spennende å se hvor lenge det tar før de blokkerer innholdet sitt for å bli vist i en iframe. Er alltid en eller annen morro-klump som finner ut at de har lyst til å sette på ekstra beskyttelses-lag på frontenden sin.

Reply:
Krysser fingrene for at det ikke skjer før vi har laget våre egne rapporter :crossed_fingers: